app/controllers/events_controller.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# Events controller
+class EventsController < ApplicationController
+  def new
```

Comment:
i can see this and i'm not logged in 
![Screenshot 2019-09-09 at 5 39 47 PM](https://user-images.githubusercontent.com/10922508/64549619-f9866d80-d328-11e9-8734-90f8ee7ad8fe.png)

- out requirement forbids that
* [ ] Logged in user can create new event **(covered by automated tests)**